### PR TITLE
Test only on all currently supported python versions

### PR DESCRIPTION
**Pull request by jdevera**

### CODE REVIEW

**Comment by borntyping** [comment]
There's still a lot of code that explicitly supports Python 2.6, so I'd prefer the tests to keep using 2.6 until that code is removed.

File: tox.ini [hunk 1]
```diff
@@ -1,11 +1,10 @@
 [tox]
 minversion=1.8.0
-envlist=py26-{test,test-colorama},py{27,34,35,36,py,py3}-{test,test-colorama,lint}
+envlist=py{27,35,36,37,38,py,py3}-{test,test-colorama,lint}
 skip_missing_interpreters=true
 
 [testenv]
 commands=
-    lint: flake8 colorlog
```
Comment (borntyping) [inline tox.ini]: Was this line removed by mistake?

Comment (jdevera) [reply tox.ini]: Yes, I'll readd.

**Comment by jdevera** [comment]
> There's still a lot of code that explicitly supports Python 2.6, so I'd prefer the tests to keep using 2.6 until that code is removed.

Sure, I'll re-add Python 2.6

**Comment by borntyping** [comment]
Thanks! :)

**Pull request merged**